src/IntroPage/index.js

```diff
@@ -117,6 +118,7 @@ export class PureIntro extends React.Component<Props> {
   render() {
     return (
       <div className="Intro">
+        <SelectUrlBooking />
```

Comment:
le'ts move it into App, it can open a different page upon opening as well

Reply:
That's true, albeit that's not even merged yet :smile: Will change :+1: